lang/cs/Org.Apache.REEF.Driver/Bridge/DriverBridgeConfiguration.cs

```diff
@@ -54,6 +54,7 @@ public class DriverBridgeConfiguration : ConfigurationModuleBuilder
         /// <summary>
         /// The event handler for requesting evaluator
         /// </summary>
+        [Obsolete]
```

Comment:
Add documentation of the `[Obsolete]` attribute: In which version was this obsoleted and what shall people use instead?